File: lib/scan_result.dart
class ScanResult {
  String result;
  String scannerType;

  ScanResult({this.result, this.scannerType});

  ScanResult.fromJson(Map<String, dynamic> json) {
    result = json['result'];
    scannerType = json['scannerType'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['result'] = this.result;
    data['scannerType'] = this.scannerType;
    return data;
  }

  @override
  String toString() {
    return 'ScanResult{result: $result, scannerType: $scannerType}';
  }
}

File: android/src/main/kotlin/com/excubivue/cubivue_scanners/scanner/models/ScanResult.kt
package com.excubivue.cubivue_scanners.scanner.models

data class ScanResult(var result: String = "", var scannerType: String = "") {

    override fun toString(): String {
        return "{" +
                "  \"result\": \"$result\"," +
                "  \"scannerType\": $scannerType" +
                "}"
    }
}